modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -3,150 +3,63 @@
+. Verify your `overwrite.cli.url` is configured to the correct URL and instead of `localhost`.
+. Reset your `oc_jobs` table.
 +
-image:configuration/files/federation-3.png[image]
+----
+mysql -u root -e "update oc_jobs set last_run=0;" owncloud;
+mysql -u root -e "update oc_jobs set last_checked=0;" owncloud;
+----
+. Navigate to **admin settings -> sharing -> Federation**
+. Add **server 1** to the trusted servers on **server 2**.
+. Add **server 2** to the trusted servers on **server 1**.
+. Now run the cron job in your ownCloud directory (for example `/var/www/owncloud/`).
 +
-You may disconnect the share at any time by clicking the btn:[trash can] icon.
-
-[[configuring-trusted-owncloud-servers]]
-== Configuring Trusted ownCloud Servers
-
-You may create a list of trusted ownCloud servers for Federation
-sharing. This allows your linked ownCloud servers to share user
-directories, and to auto-fill user names in share dialogs. If *Add
-server automatically once a federated share was created successfully* is
-enabled on your Admin page, servers will be automatically added to your
-trusted list when you create new Federation shares.
-
-You may also enter ownCloud server URLs in the *Add ownCloud Server*
-field. The yellow light indicates a successful connection, with no user
-names exchanged. The green light indicates a successful connection with
-user names exchanged. A red light means the connection failed.
-
-image:configuration/files/federation-1.png[image]
-
-== Automatically Accept Remote Shares From Trusted Servers 
-
-To automatically accept remote shares from servers in xref:configuring-trusted-owncloud-servers[the trusted servers list], when configuring Federated Cloud sharing, under menu:admin[Settings > Sharing > Federated Cloud Sharing], check the checkbox labeled "_Automatically accept remote shares from trusted servers_".
-
-image:configuration/files/sharing/federated-cloud-sharing/automatically-accept-remote-shares-from-trusted-servers.png[Automatically accept remote shares from trusted servers in ownCloud]
-
-IMPORTANT: Due to security reasons auto-accepting doesn't work when "_Add server automatically once a federated share was created successfully_" is checked.
-
-All incoming shares from remote servers are accepted automatically when:
-
-. The federation app's `autoAddServers` config option is set to `0` (default); for example:
+----
+sudo -u www-data php cron.php
+----
+. Now the check should be green
+. Sync now your users with
 +
-[source,php]
 ----
-occ config:app:set federation auto_accept_trusted --value '0'
+occ dav:sync-system-addressbook
+occ federation:sync-addressbook
 ----
-. The `federatedfilesharing` app's `auto_accept_trusted config` option is set to `yes`; for example:
+. Configure automatic acceptance of new federated shares.
 +
-[source,php]
 ----
+occ config:app:set federation auto_accept_trusted --value '0'
 occ config:app:set federatedfilesharing auto_accept_trusted --value 'yes'
```

Comment:
Those options could also be set/unset in the admin page. Worth mentioning it.

Reply:
Yes, but I found a warning message in the docs, that if you check this option in the webUI it **won't** work. So to be on the save side - better set it with occ.

Reply:
> Yes, but I found a warning message in the docs, that if you check this option in the webUI it **won't** work. So to be on the save side - better set it with occ.

Makes sense!